docs/best-practices.md

```diff
@@ -0,0 +1,14 @@
+# Best Practices
+
+Guardian CDK codifies best practices as defined in the [recommendations](https://github.com/guardian/recommendations)
+repo.
+
+Here are some best practices for using this library:
```

Comment:
Worth linking to AWS's best practice doc too? https://docs.aws.amazon.com/cdk/v2/guide/best-practices.html